frame/randomness-collective-flip/src/lib.rs

```diff
@@ -85,21 +108,20 @@ decl_module! {
 				values[index] = parent_hash;
 			});
 
-			0
-		}
-	}
+			T::DbWeight::get().reads_writes(1, 1)
+        }
+    }
+
+	/// Series of block headers from the last 81 blocks that acts as random seed material. This
+	/// is arranged as a ring buffer with `block_number % 81` being the index into the `Vec` of
+	/// the oldest hash.
+    #[pallet::storage]
+    #[pallet::getter(fn random_material)]
+    pub(super) type RandomMaterial<T: Config> =
+        StorageValue<_, Vec<T::Hash>, ValueQuery>;
```

Comment:
Some whitespace problems here.

Reply:
Fixed, hope to use rustfmt in substrate ASAP :rofl: